src/Java.Interop.Tools.Maven/IPomResolver.cs

```diff
@@ -0,0 +1,8 @@
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven;
+
+public interface IPomResolver
+{
+	Project ResolveRawProject (Artifact artifact);
```

Comment:
What is "Raw" serving in this name?  Would this be better as just `ResolveProject()`?